src/main/java/seedu/duke/PairingList.java

```diff
@@ -17,16 +17,35 @@ public class PairingList {
     private static final String LOG_DELETE_PAIR = "The following pairing(s) has been deleted from PairingList: ";
     private static final String LOG_PAIRS_WITH = "Pairs with ";
 
+    public static final String ASSERTION_ADD_PAIRING_CLIENT_ALREADY_PAIRED =
+            "Add Pairing: client already paired with property. Pairing addition unsuccessful.\n";
+
+    private static final String ASSERTION_DELETE_PAIRING_UNSUCCESSFUL =
+            "Delete Pairing: pairing deletion unsuccessful.\n";
```

Comment:
Are we using the terms "Delete Pairing" or "Unpair"
I feel it would be better if we standardise throughout.